roms_tools/setup/nesting.py

```diff
@@ -269,13 +304,28 @@ def _finalize_grid_datasets(
 
         child_grid_ds : xr.Dataset
             The child grid dataset after modifications.
+
+        Returns
+        -------
+        tuple[xr.Dataset, xr.Dataset]
+            The finalized parent and child grid datasets with longitudes wrapped to [0, 360].
+
         """
         parent_grid_ds = wrap_longitudes(parent_grid_ds, straddle=False)
         child_grid_ds = wrap_longitudes(child_grid_ds, straddle=False)
+
         return parent_grid_ds, child_grid_ds
 
     @classmethod
-    def from_file(cls, filepath: str | Path, verbose: bool = False) -> "ChildGrid":
+    def from_file(
+        cls,
+        filepath: str | Path,
+        theta_s: float | None = None,
+        theta_b: float | None = None,
+        hc: float | None = None,
+        N: int | None = None,
+        verbose: bool = False,
+    ) -> "ChildGrid":
```

Comment:
Consider expanding the name of `N` so it's purpose is clear (and so it meets naming standards).

Reply:
Oof, this syntax is inherited from ucla. Changing it would be a breaking change for the user.

Reply:
👍